.github/workflows/configure.sh

```diff
@@ -1,16 +1,26 @@
-#!/bin/bash -xe 
+#!/bin/bash -xe
 
 maker=$1
 device=$2
 
-if [ "$maker" = "cmake" ]; then
-   rm -rf build
-   mkdir -p build
+if [ "${maker}" = "cmake" ]; then
+    rm -rf build
+    mkdir -p build
 fi
 
 mydir=`dirname $0`
```

Comment:
Old style.
We should use `$()` instead of backtick \`

Reply:
Yes. I was actually unfamiliar with the `$()` syntax.